app/src/layouts/kanban/kanban.vue

```diff
@@ -159,18 +163,33 @@ const textFieldConfiguration = computed<Field | undefined>(() => {
 										:class="{ selected: selection.includes(element[primaryKeyField?.field]) }"
 										@click="onClick({ item: element, event: $event })"
 									>
-										<div v-if="element.title" class="title">{{ element.title }}</div>
+										<div v-if="element.title" class="title">
+											<render-display
+												v-if="titleFieldConfiguration"
+												:collection="collection"
+												:value="element.title"
+												:type="titleFieldConfiguration.type"
+												:field="layoutOptions?.titleField"
+												:display="titleFieldConfiguration.meta?.display"
+												:options="titleFieldConfiguration.meta?.display_options"
+												:interface="titleFieldConfiguration.meta?.interface"
+												:interface-options="titleFieldConfiguration.meta?.options"
+											/>
+										</div>
 										<img v-if="element.image" class="image" :src="element.image" />
-										<render-display
-											v-if="element.text && textFieldConfiguration"
-											:collection="collection"
-											:value="element.text"
-											:type="textFieldConfiguration.type"
-											:field="layoutOptions?.textField"
-											:display="textFieldConfiguration.meta?.display"
-											:options="textFieldConfiguration.meta?.options"
-											:interface="textFieldConfiguration.meta?.interface"
-										/>
+										<div class="text">
+											<render-display
+												v-if="element.text && textFieldConfiguration"
+												:collection="collection"
+												:value="element.text"
+												:type="textFieldConfiguration.type"
+												:field="layoutOptions?.textField"
+												:display="textFieldConfiguration.meta?.display"
+												:options="textFieldConfiguration.meta?.display_options"
+												:interface="textFieldConfiguration.meta?.interface"
+												:interface-options="textFieldConfiguration.meta?.options"
+											/>
+										</div>
```

Comment:
With the above in mind of combining the fieldConfigurations into a single computed var, then we could render both of these with a single v-for loop 🤔 

The <img> currently sits between them, but I think the CSS is such that we can move it below without affecting the layout.

Reply:
I don’t think a for-loop makes sense here, at the same time I agree that the elements look the same. That’s why I refactored the computed property to pass the `<render-display>` options via `v-bind`.